### PR TITLE
chore: taxonomy week 42

### PR DESCRIPTION
**Pull request by aleene**
### What
some trace detection phrases

### CODE REVIEW

File: lib/ProductOpener/Ingredients.pm [hunk 1]
```diff
@@ -188,16 +188,16 @@ my %may_contain_regexps = (
...
+		"peut également contenir|[P|p]eut contenir|qui utilise|utilisant|qui utilise aussi|qui manipule|manipulisant|qui manipule aussi|traces possibles|traces d'allergènes potentielles|trace possible|traces potentielles|trace potentielle|traces éventuelles|traces eventuelles|trace éventuelle|trace eventuelle|traces|trace|Traces éventuelles de|Peut contenir des traces de",
 	hr =>
 		"mogući sadržaj|mogući tragovi|može sadržavati|može sadržavati alergene u tragovima|može sadržavati tragove|može sadržavati u tragovima|može sadržati|može sadržati tragove|proizvod može sadržavati|proizvod može sadržavati tragove",
 	is => "getur innihaldið leifar|gæti innihaldið snefil|getur innihaldið",
 	it =>
-		"Pu[òo] contenere tracce di|pu[òo] contenere|che utilizza anche|possibili tracce|eventuali tracce|possibile traccia|eventuale traccia|tracce|traccia",
+		"Pu[òo] contenere tracce di|pu[òo] contenere|che utilizza anche|possibili tracce|eventuali tracce|possibile traccia|eventuale traccia|tracce|traccia|Può contenere",
```
Comment (stephanegigandet) [inline lib/ProductOpener/Ingredients.pm:196]: should be already covered by 2nd entry

**Review by stephanegigandet — APPROVED** [review]
Approved, but can you remove things like [m|M] they should not be needed. If you find an ingredient list where they are needed, could you send it to me? We might have a bug somewhere.

File: lib/ProductOpener/Ingredients.pm [hunk 2]
```diff
@@ -177,7 +177,7 @@ my $separators = qr/($stops\s|$commas|$separators_except_comma)/i;
 my %may_contain_regexps = (
 
 	en =>
-		"it may contain traces of|possible traces|traces|may also contain|also may contain|may contain|may be present",
+		"it may contain traces of|possible traces|traces|may also contain|also may contain|[m|M]ay contain|may be present|Produced in a factory handling",
```
[the remark this answers is not in the document]
Comment (aleene) [reply lib/ProductOpener/Ingredients.pm]: There might be an issue here. But I revert for the moment, Maybe add a test for case sensitive to be sure

File: lib/ProductOpener/Ingredients.pm (hunk 1 above)
Comment (aleene) [reply lib/ProductOpener/Ingredients.pm]: I revert, but might be a case sensitive issue.

**Review by stephanegigandet — APPROVED** [review]
Thanks!

**Pull request merged**